app/views/comments/index.html.erb

```diff
@@ -18,7 +18,7 @@
         <td><%= comment.body %></td>
         <td><%= link_to 'Show', comment %></td>
         <td><%= link_to 'Edit', edit_comment_path(comment) %></td>
-        <td><%= link_to 'Destroy', comment, data: { turbo_method: :delete }, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to 'Destroy', comment, data: { turbo_method: :delete, confirm: 'Are you sure?' } %></td>
```

Comment:
I wonder why `destroy` function doesn't work in this app. In mine as well. I tried to find an error, but..

Reply:
Which page are you referring to? This destroy link is on the comments index page (/comments route), which isn't a valid route (see config/routes.rb), so it will never even be visible.

Reply:
I think this is just a hold over from before I removed the comments/index route, so we can just ignore this file.